benchopt/tests/test_cli.py

```diff
@@ -67,7 +67,7 @@ def test_benchopt_run(self):
         with CaptureRunOutput() as out:
             run([str(DUMMY_BENCHMARK_PATH), '-l', '-d', SELECT_ONE_SIMULATED,
                  '-f', SELECT_ONE_PGD, '-n', '1', '-r', '1', '-p', '0.1',
-                 '--no-plot'], 'benchopt', standalone_mode=False)
+                 ], 'benchopt', standalone_mode=False)
```

Comment:
If you do this, you need to clean up after the test :)
Thus here we need to get from the output the name of the HTML page, and then remove it if it exists.

See how it is done for `csv` in `CaptureRunOutput`.